src/Pipfile

```diff
@@ -33,6 +33,7 @@ requests = "==2.21.0"
 sendgrid = "==6.0.4"
 # Hack: We are using this to specify App Engine packages,
 # since there is no support for custom package sets.
+google-cloud-secret-manager = "==2.18.0"
```

Comment:
nit: this should come before the comment. The comment is referring to the dev-packcages bit.